doc/admin-guide/configuration/transparent-proxy.en.rst

```diff
@@ -1,5 +1,7 @@
 .. _transparent-proxy:
 
+.. include:: ../../common.defs
+
```

Comment:
Looks like the docs fail to build because the admin guide cannot access the _transparent-proxy link:

https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/439/console

```
/home/jenkins/workspace/Github_Builds/docs/src/doc/admin-guide/introduction.en.rst:69:Failed to create a cross reference. A title or caption not found: transparent-proxy
```

 I think this will be fixed if we swap these two lines (that is, put the _transparent_proxy link after the include directive instead of before)..